Dockerfile

```
@@ -0,0 +1,55 @@
+FROM ubuntu:xenial as app
+
+# System requirements.
+RUN apt-get update && apt-get upgrade -qy
+RUN apt-get install -qy \
+	git-core \
+	language-pack-en \
+	python3.5 \
+	python3-pip \
+	python3.5-dev \
+	libmysqlclient-dev \
+	libssl-dev
+RUN pip3 install --upgrade pip setuptools
+RUN rm -rf /var/lib/apt/lists/*
+
+# Python is Python3.
+RUN ln -s /usr/bin/pip3 /usr/bin/pip
+RUN ln -s /usr/bin/python3 /usr/bin/python
+
+# Use UTF-8.
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+ENV DJANGO_SETTINGS_MODULE registrar.settings.production
```

Style nit: this seems separate from the block of UTF-8-related commands.
```suggestion

ENV DJANGO_SETTINGS_MODULE registrar.settings.production
```